workflow/cron/operator.go

```diff
@@ -108,14 +109,25 @@ func getWorkflowObjectReference(wf *v1alpha1.Workflow, runWf *v1alpha1.Workflow)
 }
 
 func (woc *cronWfOperationCtx) persistUpdate() {
-	data, err := json.Marshal(map[string]interface{}{"status": woc.cronWf.Status})
+	woc.patch(map[string]interface{}{"status": woc.cronWf.Status})
+}
+
+func (woc *cronWfOperationCtx) persistUpdateActiveWorkflows() {
+	woc.patch(map[string]interface{}{"status": map[string]interface{}{"active": woc.cronWf.Status.Active}})
+}
+
+func (woc *cronWfOperationCtx) patch(patch map[string]interface{}) {
+	data, err := json.Marshal(patch)
```

Comment:
I found I nice pattern for having a func which does not return errors easier to write:

```golang
err:=func() error {
//do stuff
}()
if err!= nil {
log.WithError(err).Error("stuff failed"
}
```
```

Reply:
I actually prefer the way it is now as it's easier to read. Wrapping the entire function in an anonymous function might give the impression of a goroutine or similar. Moreover it doesn't fit the patter we use across the code already. I don't feel too strongly about this and could use the patter if you wanted